front-end/src/Services/API.js

```diff
@@ -69,6 +69,20 @@ const API = {
         .catch(() => dispatch(errorProject(true)));
     };
   },
+  put(path, data, receiveAction) {
+    const url = `https://exadelcats.herokuapp.com/${path}`;
```

Comment:
Haven't you done some constant for server domain to quick changing it if needed?